modules/using-assisted-installer-oci-agent-iso.adoc

```diff
@@ -4,16 +4,21 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="using-assisted-installer-oci-agent-iso_{context}"]
-= Using the Assisted Installer to generate an OCI-compatible discovery ISO image
+= Using the {ai-full} to generate an {oci}-compatible discovery ISO image
 
 Generate a discovery ISO image and upload the image to {oci-first}, so that the agent can perform hardware and network validation checks before you install an {product-title} cluster on {oci}.
 
+From the {oci} web console, you must create the following resources:
+
+* A compartment for better organizing, restricting access, and setting usage limits to {oci} resources.
+* An object storage bucket for safely and securely storing the discovery ISO image. You can access the image at a later stage for the purposes of booting the instances, so that you can then create your cluster.
```

Comment:
> a later stage

Later stage of what? Or, perhaps this is understood as part in the OCI context??

Reply:
Yes, this becomes clearer further on in the integrated workflow. It's now in the Oracle docs so general references are safer for better doc maintainability on our side.

![Screenshot from 2024-05-01 15-33-35](https://github.com/openshift/openshift-docs/assets/57954076/6b39833d-9784-4119-af23-19f3df976945)